### DISCUSSION TITLE
I have a question about Arch based distributions

### DISCUSSION THREAD

**Discussion by YuvanMichaelVivenzi** [evilsocket/opensnitch#634] [Releases]
Hello, I am Yuvan Michael. I use Manjaro but couldn't find any mention of installing this on manjaro/arch in the releases.

**Comment by gustavo-iniguez-goya** [comment]
hey @YuvanMichael ,

It's not mentioned because we don't generate packages for those distributions and derivatives. Once the AUR packages maintainers update the packages, you'll be notified about the new release by your package manager.

If opensnitch were packaged for major distributions probably we wouldn't need to generate deb/rpm packages either.

**Comment by YuvanMichaelVivenzi** [comment]
So, should I just go with the opensnitch-git AUR?

**Comment by YuvanMichaelVivenzi** [comment]
Sorry for the late reply.

**Comment by YuvanMichaelVivenzi** [comment]
I'm planning to use this. https://aur.archlinux.org/packages/opensnitch-git

**Reply by gustavo-iniguez-goya** [reply]
yes @YuvanMichael , that package will contain latest changes from this repository (which might have bugs).

If you want something more stable, use: https://aur.archlinux.org/packages/opensnitch

**Comment by YuvanMichaelVivenzi** [comment]
Thank you.

**Comment by YuvanMichaelVivenzi** [comment]
I'm having errors in making open snitch.

**Comment by YuvanMichaelVivenzi** [comment]
Update: Using manjaro's add or remove software tool. (Pamac). Worked. :)

**Comment by YuvanMichaelVivenzi** [comment]
The daemon's not starting. I only have opensnitch-ui open.

**Reply by gustavo-iniguez-goya** [reply]
You need to start it manually I think: `systemctl enable opensnitch; systemctl start opensnitch`

**Comment by YuvanMichaelVivenzi** [comment]
Error not found.
